src/components/MovieCard/MovieCard.scss

```
@@ -1 +1,3 @@
-// 
+.card-image img {
+  object-fit: contain;
+}
```

```suggestion
}

```
Пустий рядок в кінці файлу має бути